### DISCUSSION TITLE
Possibilità di filtrare utilizzando la data di pagamento del issuedDocuments

### DISCUSSION THREAD

**Discussion by NX-Massimo-Frangiamore** [fattureincloud/api#234] [Ideas]
Possibilità di filtrare utilizzando la data di pagamento del issuedDocuments

**Comment by emajo** [comment]
Ciao @NX-Massimo-Frangiamore, credo che sarebbe un filtro di difficile implementazione e utilizzo in quanto un documento può avere all'interno più tranches di pagamento, e quindi più date di pagamento, per la maggiorparte degli utilizzi è sufficiente filtrare sul campo next_due_date, se le vengono in mente degli use cases per i quali non è sufficente filtrare su quel campo li condivida pure e ne valuteremo la fattibilità.

Grazie,
Emanuele.

**Reply by NX-Massimo-Frangiamore** [reply]
l'use cases è di recuperare tutte le fatture saldate dopo una data x, pensavo che poteva essere utilizzato il campo updated_at ma da una verifica risulta che tale campo non viene aggiornato una volta saldata la fattura
